mealie/schema/openai/recipe_ingredient.py

```diff
@@ -72,8 +73,8 @@ class OpenAIIngredient(OpenAIBase):
         ),
     )
 
-    @field_validator("quantity")
-    def coerce_none_qty(cls, v: float | None) -> float:
+    @field_validator("confidence", "quantity", mode="before")
```

Comment:
was this supposed to change `quantity` to `confidence`? Aren't there two confidence validators now?